conans/client/rest/rest_client_common.py

```diff
@@ -175,7 +169,6 @@ def get_json(self, url, data=None):
         headers = self.custom_headers
         if data:  # POST request
             headers.update({'Content-type': 'application/json',
-                            'Accept': 'text/plain',
```

Comment:
This was very weird, it seems the `text/plain` was just being overwritten by json (later in declaration)